### PR TITLE
Center the database message 

### PR DESCRIPTION
**Pull request by khalifan-kfan**
# Description

Putting the the message provided when a user has not created any database yet in the center of the page 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Trello Ticket ID

https://trello.com/c/DSPo0JLC

## How Can This Been Tested?

On the develop branch, login and click on the database in the sidebar, if you have no databases it displays a message to indicate that but the message is at the top on the page, when you switch to this branch you should test if the text is at the center of the screen.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

# Screenshots

alignment before
center align

### CODE REVIEW

**Review by StephenAraka — COMMENTED** [review]
good job @khalifan-kfan,
@neelxie I guess we can meet to review this together?

**Review by rhodinemma — APPROVED** [review]
thanks @khalifan-kfan

**Review by neelxie — APPROVED** [review]

**Review by fel-w — APPROVED** [review]
Lgtm

File: src/components/DatabaseList/DatabaseList.module.css [hunk 1]
```diff
@@ -92,6 +92,7 @@
   flex-grow: 1;
   display: flex;
   font-size: 16px;
+  padding-top: 15%;
```
Comment (kawerewagaba) [inline src/components/DatabaseList/DatabaseList.module.css:95]: we shall have to sync on this

**Review by kawerewagaba — CHANGES_REQUESTED** [review]
will need you to explain some things

**Review by rhodinemma — APPROVED** [review]

**Review by rhodinemma — APPROVED** [review]

**Review by fel-w — APPROVED** [review]

**Review by neelxie — APPROVED** [review]

File: src/components/DatabaseList/DatabaseList.module.css (hunk 1 above)
Comment (neelxie) [reply src/components/DatabaseList/DatabaseList.module.css]: We had a sync on this and the problem was that the container didn't have any height at all and we gave it that.

Comment (neelxie) [reply src/components/DatabaseList/DatabaseList.module.css]: We had a sync on this and the problem was that the container didn't have any height at all and we gave it that.

**Review by StephenAraka — APPROVED** [review]

**Review by MutegekiHenry — APPROVED** [review]
Looks good

**Comment by kawerewagaba** [comment]
will merge based on majority vote

**Pull request merged**